adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/PlatformParameters.cs

```diff
@@ -37,6 +38,11 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
     [CLSCompliant(false)]
     public class PlatformParameters : IPlatformParameters
     {
+        static PlatformParameters()
+        {
+            ModuleInitializer.EnsureModuleInitialized();
```

Comment:
>ModuleInitializer.EnsureModuleInitialized(); [](start = 12, length = 44)

I find it weird that it goes here... I realize this is what ADAL does today but still, to me this is very odd and not intuitive

Reply:
I somewhat agree. PlatformParameters is a data class and should neither log nor throw exceptions. But it is possible that a developer adds logging or exception code here. In any case, these static constructors fire once, irrespective of threads etc. so there shouldn't be a performance issue.